library/Ivoz/Provider/Domain/Service/Terminal/Provision/ProvisionSpecific.php

```diff
@@ -191,8 +191,12 @@ private function getTerminalByUrl(string $specificUrlPattern): ?TerminalInterfac
         return null;
     }
 
-    protected function extractFileExtension(string $route): string
+    protected function extractFileExtension(string|null $route): string
```

Comment:
```suggestion
    protected function extractFileExtension(?string $route): string
```

Sorry, we always use this syntax for nullable params.